### PR TITLE
chore: Refactor tables in all screen for RTL

### PR DESCRIPTION
**Pull request by iamsivin**
- Tables in all screens

### CODE REVIEW

File: app/javascript/dashboard/assets/scss/plugins/_multiselect.scss [hunk 1]
```diff
@@ -48,7 +48,7 @@
         overflow: hidden;
         text-overflow: ellipsis;
         white-space: nowrap;
-        width: 100%;
+        width: fit-content;
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/plugins/_multiselect.scss:51]: These changes will fix the width issue on the label report page.
Screenshot 2023-02-23 at 8 22 23 PM

File: app/javascript/dashboard/assets/scss/widgets/_woot-tables.scss [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 
   tbody {
     tr {
-      border-bottom: 1px solid $color-border-light;
+      border-bottom: 1px solid var(--color-border-light);
     }
 
     td {
-      padding: $space-one $space-small;
+      padding: var(--space-one) var(--space-small);
```
Comment (nithindavid) [inline app/javascript/dashboard/assets/scss/widgets/_woot-tables.scss:19]:
```suggestion
      padding: var(--space-small);
```

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue [hunk 3]
```diff
@@ -291,6 +291,24 @@ export default {
   overflow: hidden;
 }
 
+.app-rtl--wrapper {
+  .contacts-table-wrap {
+    &::v-deep {
+      .ve-table-content {
+        .ve-table-body .ve-table-body-tr .ve-table-body-td {
+          text-align: right !important;
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue:299]: Can we avoid the important?

Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue:300]: @nithindavid I tried it but not working

**Review by nithindavid — COMMENTED** [review]
image
An extra sidebar is coming in contacts table

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue (hunk 3 above)
Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue]: Done https://github.com/chatwoot/chatwoot/pull/6525/commits/b1e55901bcc1b5ce8d5116a21fdbd9604470e741

**Review by nithindavid — APPROVED** [review]

**Pull request merged**